getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,89 @@
+that you need to ensure that a user that is connected to the access
+node can access the same databases on the data nodes.
+
+> NOTE: Setting up a secure system is a complex task and this section
+> should not be read as recommending any particular security measures
+> for securing your system. The chapter contains technical
+> instructions for how to set up the different authentication methods.
+
+## Setting up Password Authentication [](distdb-auth.pass)
+
+The easiest authentication to set up is password authentication. It is
+not as strong as certificate authentication, but can be used for an
+initial setup or when you have good perimeter security. In this
+section, the focus is on setting up SCRAM SHA-256 password
+authentication. For other password authentication methods, [see the
+PostgreSQL manual][auth-password].
+
+> NOTE: The `passwd` method should not be used since it sends the
+> password in cleartext, and the `md5` is using a cryptographic hash
+> function that is no longer considered secure.
+
+### Setting up Password Authentication on the Data Node [](distdb-auth.pass.data-node)
+
+In order to set up password authentication for a user `new_user`, it
+is necessary to update the HBA (Host-Based Authentication) file on the
+data nodes to enable password authentication for `new_user`. The
+default name of the HBA file is `pg_hba.conf`. Details of [the
+`pg_hba.conf` format can be found in the PostgreSQL
+manual.][postgresql-hba].
+
+In this case, you want to give `new_user` in the `example.com` domain
+access to all databases on the data node, so add the following line to
+the `pg_hba.conf`.
+
+```
+host    all    new_user    .example.com    scram-sha-256
+```
+
+Once you have set up the authentication method on the data node, you
+need to check the configuration file (normally `postgresql.conf` in
+the data directory of the data node server) and make sure that the
+`hba_file` option is correctly set.
+
+### Setting up Passwords on the Access Node [](distdb-auth.pass.access-node)
+
+Passwords are not passed through the command line, but are added to a special
+password file. The default name of the password file is `passfile`
+located in the data directory for the PostgreSQL server, but it can be set
+in the `postgresql.conf` file for the access node using the option
+`timescaledb.passfile`. You should also set the name of the HBA file
+using the `hba_file` option:
+
+```
+timescaledb.passfile = 'pgpass.conf'
+```
+
+**Password file format.* The password file format is [described in
+PostgreSQL manual][passfile-format] and consists of lines of the
+format:
+
+```
+hostname:port:database:username:password
+```
+
+So, in order to add a password for our user `new_user` and use
+password `xyzzy`, you can create a password file with the following
+contents:
+
+```
+*:*:*:new_user:xyzzy
+```
+
+[postgresql-hba]: https://www.postgresql.org/docs/11/auth-pg-hba-conf.html
+[auth-password]: https://www.postgresql.org/docs/11/auth-password.html
```

Comment:
Let's use `current` instead of `11` here. Then we need not update with every PG release.

Reply:
Done.